### PR TITLE
less colors, add input counter, disable enter keypress

### LINKED ISSUE #65: [Add] Text input length info
The user must know before generating a postcard what is the maximum length of the message.

### LINKED ISSUE #73: [Fix] Enter button
Nothing happens when user presses 'Enter' in index.html. 
The only way he can generate a postcard is by hitting the 'Generate' web button.

### PR DESCRIPTION
**Pull request by nyotaL**
Screenshot 2020-09-23 at 12 25 03 PM
Screenshot 2020-09-23 at 12 25 30 PM
Screenshot 2020-09-23 at 12 25 40 PM
Screenshot 2020-09-23 at 12 26 15 PM
Screenshot 2020-09-23 at 2 49 04 PM

### CODE REVIEW

File: src/main/webapp/scripts/script.js [hunk 1]
```diff
@@ -11,6 +11,19 @@
...
+
+function count(input_id, counter_id, length) {
+    let el = document.getElementById(counter_id);
+    el.innerHTML = "<text>" + document.getElementById(input_id).value.length + length + "</text>";
+}
+
+let textInput = document.getElementById("input-text");
+if (textInput) {
+    textInput.addEventListener("input", function() {count("input-text", "result_user", "/100");}, false);
+    textInput.addEventListener("click", function() {textInput.value = "";}, false);
```
Comment (Krechals) [inline src/main/webapp/scripts/script.js]: Try calling count(...) directly.

File: src/main/webapp/scripts/script.js [hunk 2]
```diff
@@ -11,6 +11,19 @@
...
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
+
+
+function count(input_id, counter_id, length) {
+    let el = document.getElementById(counter_id);
+    el.innerHTML = "<text>" + document.getElementById(input_id).value.length + length + "</text>";
+}
+
+let textInput = document.getElementById("input-text");
```
Comment (Krechals) [inline src/main/webapp/scripts/script.js]: I would avoid using global variables. I explained some of the reasons here https://github.com/googleinterns/step230-2020/pull/45 (first comment).

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -50,14 +50,15 @@
       </div>
       <div class="inline">
         <form id="input-form">
-          <input type="text" class="input-text" id="input-text" name="input-text" placeholder="Enter your postcard text.."> 
+          <input type="text" class="input-text" id="input-text" maxlength="100" name="input-text" onkeydown="return (event.keyCode!=13);" placeholder="Enter your postcard text.."> 
+          <span id="result_user">0/100</span>
```
Comment (Krechals) [inline src/main/webapp/index.html]: The maximum length of a message should be 300. https://github.com/googleinterns/step230-2020/pull/60

Comment (Krechals) [reply src/main/webapp/index.html]: Also, try to use hyphen as word separator in ID. https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

File: src/main/webapp/scripts/script.js (hunk 1 above)
Comment (Krechals) [reply src/main/webapp/scripts/script.js]: Also, try to add whitespaces accordingly https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace.

**Review by Krechals — CHANGES_REQUESTED** [review]

**Review by Krechals — APPROVED** [review]
Please try to add a screenshot with the new design in the description.

File: src/main/webapp/index.html [hunk 4]
```diff
@@ -50,17 +50,18 @@
       </div>
       <div class="inline">
         <form id="input-form">
-          <input type="text" class="input-text" id="input-text" name="input-text" placeholder="Enter your postcard text.."> 
+          <input type="text" class="input-text" id="input-text" maxlength="300" name="input-text" onkeydown="return (event.keyCode!=13);" placeholder="Enter your postcard text..">
```
Comment (AndreMB0) [inline src/main/webapp/index.html]: isn't the max length of characters for the postcard 310?
this._maxMessageLength = 310;

also below

File: src/main/webapp/scripts/script.js [hunk 5]
```diff
@@ -11,6 +11,15 @@
 //  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
+
+
+if (document.getElementById('input-text')) {
+    document.getElementById('input-text').addEventListener('input', function() {
+        let el = document.getElementById('result-user');
+        el.innerHTML = "<text>" + document.getElementById('input-text').value.length + "/300" + "</text>";
```
Comment (AndreMB0) [inline src/main/webapp/scripts/script.js]: also here

**Review by AndreMB0 — CHANGES_REQUESTED** [review]
Please add a description for the PR with some screenshots

File: src/main/webapp/index.html (hunk 4 above)
Comment (nyotaL) [reply src/main/webapp/index.html]: hmm Andrei told me it's 300. I'll check on that

**Comment by nyotaL** [comment]
Screenshot 2020-09-23 at 12 25 03 PM
Screenshot 2020-09-23 at 12 25 30 PM
Screenshot 2020-09-23 at 12 25 52 PM
Screenshot 2020-09-23 at 12 26 15 PM

**Review by AndreMB0 — APPROVED** [review]

**Pull request merged**